packages/app/src/paths.ts

```diff
@@ -1,5 +1,11 @@
+import { decodeBase64Unicode } from '@packages/frontend-shared/src/utils/base64'
+
 export function getPathForPlatform (posixPath: string) {
-  if (Cypress.platform === 'win32') return posixPath.replaceAll('/', '\\')
+  // @ts-ignore
+  const cy = window.Cypress
+  const platform = cy?.platform || JSON.parse(decodeBase64Unicode(window.__CYPRESS_CONFIG__.base64Config)).platform
+
+  if (platform === 'win32') return posixPath.replaceAll('/', '\\')

```

Comment:
Why do we need this on the front-end? I thought we handled all path related things on the B/E?

Reply:
This is for populating the right default file name. Since I don't have a windows machine I didn't want to touch it